esmvaltool/recipes/recipe_python_test.yml

```diff
@@ -0,0 +1,39 @@
+---
```

Comment:
This file should probably not be here

Reply:
You're right, I will remove it then, thanks

Reply:
Done!